modules/redis/redis-worker.c

```diff
@@ -277,17 +352,19 @@ redis_worker_connect(LogThreadedDestWorker *s)
 }
 
 
-LogThreadedDestWorker *redis_worker_new(LogThreadedDestDriver *owner, gint worker_index)
+LogThreadedDestWorker *redis_worker_new(LogThreadedDestDriver *o, gint worker_index)
 {
   RedisDestWorker *self = g_new0(RedisDestWorker, 1);
+  RedisDriver *owner = (RedisDriver *) o;
 
-  log_threaded_dest_worker_init_instance(&self->super, owner, worker_index);
+  log_threaded_dest_worker_init_instance(&self->super, o, worker_index);
 
   self->super.thread_init = redis_worker_thread_init;
   self->super.thread_deinit = redis_worker_thread_deinit;
   self->super.connect = redis_worker_connect;
   self->super.disconnect = redis_worker_disconnect;
-  self->super.insert = redis_worker_insert;
+  self->super.insert = owner->super.batch_lines > 0 ? redis_worker_insert_batch : redis_worker_insert;
```

Comment:
The `batch-lines()` option is missing from the `redis()` modules.
Please add `threaded_dest_driver_batch_option` to the grammar.

Reply:
Is it guaranteed that this constructor runs only after batch_lines is set (after parsing and setting all config options)?